modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc

```diff
@@ -0,0 +1,41 @@
+= Use with GCP Pay As You Go
+:description: Subscribe to Redpanda in Google Cloud Marketplace with pay-as-you-go billing, and cancel anytime.
+
+Subscribe to Redpanda through Google Cloud Marketplace to quickly provision Dedicated Cloud clusters. With a usage-based pay as you go subscription, you only pay for what you use and can cancel anytime. 
+
+[IMPORTANT]
+====
+When you sign up for Redpanda through Google Cloud Marketplace, you can only create clusters on GCP. 
+====
+
+== Sign up in Google Cloud Marketplace
+
+. On https://console.cloud.google.com/marketplace[Google Cloud Marketplace^], search for and select "Redpanda SaaS Offering".
+
+. On the Redpanda SaaS Offering overview page, click **Subscribe**.
++
+[NOTE]
+====
+If you don't have a billing account associated with your project, you're prompted to enable billing to link the subscription with a billing account.
+====
+
+. On the Order Summary page, the **Pay As You Go** plan using Redpanda consumption units is automatically selected. 
+* Read and accept the terms.
+* Click **Subscribe**.
+
+. On the Redpanda page: 
+* For **Email**, enter your email address to register with Redpanda.
+* For **Organization name**, enter a name for your new organization connected through Google Cloud Marketplace. Redpanda organizations contain all resources, including clusters and networks. You can rename the organization name later in Redpanda Console. 
```

Comment:
@c4milo can we change the organization name in Cloud?

Reply:
I think not, I don't see it ... but will wait for someone from UI to confirm

Reply:
cc @malinskibeniamin @yougotashovel

Reply:
We can manually in Auth0, but customers cannot.  Not in our roadmap but it's technically plausible.

Reply:
Removing: "You can rename the organization later in Redpanda Console."